.travis.yml

```diff
@@ -13,8 +13,16 @@ install:
   - source config/settings.test
   - export PYTHONPATH=.
 
+services:
+  - postgresql
```

Comment:
Can we specify we should use `postgresql 9.3` (that's what we run on deployment). Right now, travis is using `9.4`.